librz/bin/bin.c

```diff
@@ -605,15 +605,16 @@ RZ_API void rz_bin_set_baddr(RzBin *bin, ut64 baddr) {
  *
  * \param o Reference to the \p RzBinObject instance
  * \param off Address to search
- * \param va When 0 the offset \p off is considered a physical address, otherwise a virtual address
+ * \param va When false the offset \p off is considered a physical address, otherwise a virtual address
  * \return Pointer to a \p RzBinSection containing the address
  */
-RZ_API RZ_BORROW RzBinSection *rz_bin_get_section_at(RzBinObject *o, ut64 off, int va) {
+RZ_API RZ_BORROW RzBinSection *rz_bin_get_section_at(RZ_NONNULL RzBinObject *o, ut64 off, bool va) {
+	rz_return_val_if_fail(o, NULL);
+
 	RzBinSection *section;
 	void **iter;
 	ut64 from, to;
 
-	rz_return_val_if_fail(o, NULL);
 	// TODO: must be O(1) .. use sdb here
```

Comment:
Remove this TODO as well then, since there are no plans to move them to SDB anymore